custom_components/tuya_local/devices/kogan_lx10_vacuum.yaml

```diff
@@ -0,0 +1,259 @@
+    - id: 116
+      name: total_clean_time
+      type: integer
+      unit: min
+    - id: 117
+      name: total_clean_area
+      type: integer
+      unit: m²
+    - id: 118
+      name: total_clean_count
+      type: integer
+    - id: 119
+      name: side_brush_time
+      type: integer
+      unit: s
+    - id: 120
+      name: main_brush_time
+      type: integer
+      unit: s
+    - id: 121
+      name: filter_time
+      type: integer
+      unit: s
+    - id: 122
+      name: robot_fault
+      # TODO: correctly parse the bitfield
+      type: boolean # should result in non-zero showing as true
+      # type: bitfield
+      # "values": "{\"label\":[\"1\",\"2\",\"3\",\"4\",\"5\",\"6\",\"7\",\"8\",\"9\",\"10\",\"11\",\"12\",\"13\",\"14\",\"15\",\"16\",\"17\",\"18\",\"19\",\"20\",\"21\",\"22\",\"23\",\"24\",\"25\",\"26\",\"27\",\"28\",\"29\",\"30\"],\"maxlen\":30}"
+      # mapping:
+      #   - dps_val: 1
+      #     value: cliff
+      #   - dps_val: 2
+      #     value: imp
+      #   - dps_val: 4
+      #     value: whl
+      #   - dps_val: 8
+      #     value: brush
+      #   - dps_val: 16
+      #     value: fan
+      #   - dps_val: 32
+      #     value: roller_brush
+      #   - dps_val: 64
+      #     value: low_power
+      #   - dps_val: 128
+      #     value: give_up
+      #   - dps_val: 256
+      #     value: no_dust
+      #   - dps_val: 512
+      #     value: "none"
+    - id: 123
+      name: path_comm
+      optional: true
+      type: string
+    - id: 124
+      name: cmd_comm
+      optional: true
+      type: string
+    - id: 125
+      name: request_data
+      optional: true
+      type: string
+    - id: 126
+      name: comm_flag
+      optional: true
+      type: string
+    - id: 127
+      name: comm_raw
+      optional: true
+      type: string
+    - id: 128
+      name: message_report
+      type: string
+      mapping:
+        # TODO: figure out the meanings of these
+        - dps_val: '0'
+          value: '0'
+        - dps_val: '1'
+          value: '1'
+        - dps_val: '2'
+          value: '2'
+        - dps_val: '3'
+          value: '3'
+        - dps_val: '4'
+          value: '4'
+        - dps_val: '5'
+          value: '5'
+        - dps_val: '6'
+          value: '6'
+        - dps_val: '7'
+          value: '7'
+        - dps_val: '8'
+          value: '8'
+        - dps_val: '9'
+          value: '9'
+        - dps_val: '10'
+          value: '10'
+        - dps_val: '11'
+          value: '11'
+        - dps_val: '12'
+          value: '12'
+        - dps_val: '13'
+          value: '13'
+        - dps_val: '14'
+          value: '14'
+        - dps_val: '15'
+          value: '15'
+        - dps_val: '16'
+          value: '16'
+        - dps_val: '17'
+          value: '17'
+        - dps_val: '18'
+          value: '18'
+        - dps_val: '19'
+          value: '19'
+        - dps_val: '20'
+          value: '20'
+    - id: 129
+      name: reset_map
```

Comment:
Maybe split into a separate button for completeness, but the map is cloud based and won't be available, so not really useful